lib.php

```diff
@@ -214,15 +209,27 @@ public function get_links($linkarray) {
             // Get turnitin submission details.
             $plagiarismfile = plagiarism_turnitinsim_submission::get_submission_details($linkarray);
 
+            $submission = null;
```

Comment:
Can you move that up 4 lines please? Just to be next to all the other initialisations! 

Sorry for being pedantic! 😄